### PR TITLE
support for manually file stream reader

### PR DESCRIPTION
**Pull request by Ali-YousefiTelori**
I need to read file stream manually so I created this method.

### CODE REVIEW

**Comment by karb0f0s** [comment]
If you wish to participate please follow [CONTRIBUTING](https://github.com/TelegramBots/Telegram.Bot/blob/master/CONTRIBUTING.md) recommendations:
- submit your PR to develop branch
- your PR should include updated changelog in CHANGELOG.md file
- you should provide suitable test cases for your PR

Feel free to join our [group chat](https://t.me/tgbots_dotnet).

File: src/Telegram.Bot/TelegramBotClient.cs [hunk 1]
```diff
@@ -733,6 +733,26 @@ await response.Content.CopyToAsync(destination)
...
+            if (string.IsNullOrWhiteSpace(filePath) || filePath.Length < 2)
+            {
+                throw new ArgumentException("Invalid file path", nameof(filePath));
+            }
+
+            var fileUri = new Uri($"{BaseFileUrl}{_token}/{filePath}");
+
+            var response = _httpClient
+                .GetAsync(fileUri, HttpCompletionOption.ResponseHeadersRead, cancellationToken)
+                .Result;
```
Comment (karb0f0s) [inline src/Telegram.Bot/TelegramBotClient.cs]:
```csharp
.ConfigureAwait(false);
```

File: src/Telegram.Bot/TelegramBotClient.cs [hunk 2]
```diff
@@ -733,6 +733,26 @@ await response.Content.CopyToAsync(destination)
...
+        )
+        {
+            if (string.IsNullOrWhiteSpace(filePath) || filePath.Length < 2)
+            {
+                throw new ArgumentException("Invalid file path", nameof(filePath));
+            }
+
+            var fileUri = new Uri($"{BaseFileUrl}{_token}/{filePath}");
+
+            var response = _httpClient
```
Comment (karb0f0s) [inline src/Telegram.Bot/TelegramBotClient.cs]: it is an async method, request should be awaited
```csharp
var response = await _httpClient
```

**Comment by Ali-YousefiTelori** [comment]
I cannot join to your telegram group, My telegram desktop show me an error for that link.hovewer telegram web show me "Not found", My UserName is @Ali_Visual_Studio

**Comment by karb0f0s** [comment]
@Ali-YousefiTelori if you're from Iran try using VPN or a proxy. The group isn't removed. All users from Iran have issues with public channels and supergroups.

**Comment by Ali-YousefiTelori** [comment]
I trying with VPN but not work show me "Username @tgbots_dotnet not found" but shows me group info when I share link on telegram, I don't see any group link like that, because we cannot make public link for groups.what is that? I'm using telegram desktop.

**Comment by karb0f0s** [comment]
it is not possible to commit to master branch

**Pull request closed without merging**